src/main/java/com/pashonokk/dvdrental/entity/Language.java

```diff
@@ -0,0 +1,49 @@
+package com.pashonokk.dvdrental.entity;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+import java.time.LocalDate;
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+
+@Entity
+@NoArgsConstructor
+@Getter
+@Setter
+@ToString
+public class Language {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(unique = true, nullable = false, updatable = false)
+    private String name;
+    private LocalDate lastUpdate;
+    @OneToMany(mappedBy = "language")
+    @Setter(AccessLevel.PRIVATE)
+    private Set<Film> films = new HashSet<>();
+
+    public void addFilm(Film film) {
```

Comment:
do we need to add Film to language? I gues it could be grate when we add film also set language as well